appwrite/website
# Add Deno vs Bun blog post

## What does this PR do?

Adds Deno vs Bun blog post

## Test Plan

- Run the app and navigate to `/blog/post/deno-vs-bun-javascript-runtime
- Confirm that blog post and assets are displayed correctly
